pkg/ebpf/c/bpf_tracing_custom.h

```diff
@@ -16,6 +16,26 @@
 #define PT_REGS_PARM9(x) PT_REGS_STACK_PARM(x,3)
 #define PT_REGS_PARM10(x) PT_REGS_STACK_PARM(x,4)
 
+#define DI_REGISTER_0 ax
```

Comment:
Is there any specific reason behind the ordering?
For me, I would expect the registers to be ordered by the architecture's calling convention, for example.

Reply:
Yea it is the x86 ABI: https://refspecs.linuxfoundation.org/elf/x86_64-abi-0.95.pdf#page=56

Reply:
It is the mapping between DWARF register numbers and architecture register names. Can you add the link you posted as a comment?

Reply:
maybe naming them `DWARF_REGISTER_X` would make it clearer?